src/batch-client/index.ts

```diff
@@ -1789,7 +1789,7 @@ export class ZimbraBatchClient {
 			(this.sessionHandler && this.sessionHandler.readSessionId()),
 		origin: this.origin,
 		userAgent: this.userAgent,
-		...(this.notifier.getSequenceNumber() && {
+		...(typeof this.notifier.getSequenceNumber() !== 'undefined' && {
```

Comment:
why this check?

Reply:
When we reset `sessionSeq` to 0, it's value will be 0 and In that case it will be `false` here and won't pass `sessionSeq`.
So added this check to avoid above mentioned scenario.

Reply:
this will always return some numeric value right?

Reply:
Yes. And it mostly it gets it's value from [response headers](https://github.com/Zimbra/zm-api-js-client/blob/master/src/batch-client/notifier/index.ts#L47) (`header.context.notify.0`). It is possible that `header.context.notify.0` gets `undefined`. But we set `this.sequenceNo` if header sequence is not `undefined`.